splight_cli/hub/component/exceptions.py

```diff
@@ -47,5 +47,4 @@ def __str__(self) -> str:
         return self._msg
 
 
-class BuildError(Exception):
-    ...
+class BuildError(Exception): ...
```

Comment:
```suggestion
class BuildError(Exception):
    pass
```

This change is to avoid conflicts between `ruff` and `black`